src/super_gradients/recipes/dataset_params/cifar100_dataset_params.yaml

```diff
@@ -0,0 +1,54 @@
+-batch_size: 256 # batch size for trainset
+-val_batch_size: 512 # batch size for valset in DatasetInterface
+
+# TODO: REMOVE ABOVE, HERE FOR COMPATIBILITY UNTIL WE REMOVE DATASET_INTERFACE
+
+train_dataset_params:
+  root: /data/cifar100
+  train: True
+  transform:
+    _target_: torchvision.transforms.Compose
+    transforms:
+      - _target_: torchvision.transforms.RandomCrop
+        size: 24
+        padding: 4
+      - _target_: torchvision.transforms.RandomHorizontalFlip
+      - _target_: torchvision.transforms.ToTensor
+      - _target_: torchvision.transforms.Normalize
+        mean: (0.4914, 0.4822, 0.4465)
+        std: (0.2023, 0.1994, 0.2010)
+  target_transform: null
+  download: True
+
+train_dataloader_params:
+  batch_size: 256
+  num_workers: 8
+  batch_sampler: True
+  sampler:
+    InfiniteSampler:
```

Comment:
This is not the default sampler thats used for cigar....we only use infinite sampler for YoloX training,
Just leave it blank and the sampler will be deririved by wether we're in DDP or not,